test/system/post_test.rb

```diff
@@ -0,0 +1,36 @@
+require "application_system_test_case"
+# https://guides.rubyonrails.org/testing.html#implementing-a-system-test
+
+class PostTest < ApplicationSystemTestCase
+
+  def setup
+    activate_authlogic
+  end
+
+  test 'posting from the editor' do
+    visit '/'
+
+    click_on 'Login'
+    fill_in("username-login", with: "Bob")
+    fill_in("password-signup", with: "secretive")
+    click_on "Log in"
+
+    visit '/post'
+
+    fill_in("Title", with: "My new post")
+    
+    el = find(".wk-wysiwyg") # rich text input
+    el.set("All about this interesting stuff")
+
+    find('.ple-publish').click
+    # find('.ple-publish').click # may have to do it twice if it prompts for an image
+
+    assert_response :redirect
```

Comment:
Hmm, here... `NoMethodError:         NoMethodError: undefined method `response_code' for nil:NilClass`

Reply:
line 26 is not right, we shouldn't have to click twice. Not sure why this isn't working...

Reply:
Hmm. i used Capycorder to try this but it didn't get everything:

```
it 'SHOULDDOSOMETHING' do
  visit('/post')
  page.should have_selector('.wk-wysiwyg')
  page.should have_selector('html  > body > .container > .row > .pl-editor > .ple-content > .ple-module-title.ple-module.row > .ple-module-content.col-lg-9:nth-child(1) > .form-control.input-lg')
  fill_in('undefined', :with => 'title title')
  page.should have_selector('.ple-publish.btn.btn-lg.btn-primary.float-right')
end
```

Reply:
But in fact on stable, posting is not working, so maybe this is just because the functionality is broken?

Reply:
i see a 500 on https://stable.publiclab.org/post,

```

Name
--
```

Reply:
Is this 500 possibly due to email not sending? I thought we'd resolved this but who knows... hmm. We may need to try to pull the logs here since this is not failing in tests.

Reply:
OK, think I resolved the 500 on stable... testing....

Reply:
Yes, it's resolved on stable. Still not here, though.

Reply:
OK, I realized that it's not doing a real redirect... it's JS sending us to a new page. So, I'm using a snippet from https://stackoverflow.com/questions/20243208/how-can-i-test-in-capybara-that-a-page-has-not-reloaded-javascript-onclick-in to show that `assert_page_reloads` and then just going forward with the next steps!